lib/rajska.ex

```diff
@@ -130,6 +131,8 @@ defmodule Rajska do
         |> get_current_user()
         |> has_user_access?(scoped_struct, rule)
       end
+      
+      def default_authorize(_context, _object), do: unquote(default_authorize)
```

Comment:
I think we don't need to pass these arguments:

```suggestion
      def default_authorize, do: unquote(default_authorize)
```

Reply:
That's true in the case of the default implementation, but we had to use it to allow Relay types to all:

```ex
def default_authorize(_context, %{definition: Absinthe.Relay.Connection.Types}), do: :all
```

once we finalize the tests I will add this to the docs.

Reply:
Alright, thanks for the clarification